recipes/noiseflow/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set org = "shakeflow" %}
+{% set name = "noiseflow" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 6bb97817c0bc9f2e5490e2969233bdfbcb1c6c70f236a5e6d156031a3f686656
+
+build:
+  number: 0
+  script: 
+    - git clone https://github.com/xtensor-stack/xtensor-fftw extern/xtensor-fftw  # [unix]
+    - git clone https://github.com/kfrlib/kfr extern/kfr  # [unix]
```

Comment:
You shouldn't need to clone these packages using git. xtensor-fftw is already available as a conda-forge package. If you want to link against kfr, please add a recipe to this PR for libkfr.